.sync.yml

```diff
@@ -13,32 +16,27 @@
   branches:
     - release
 
+.rubocop.yml:
+   default_configs:
+    inherit_from: .rubocop_todo.yml
+    require:
+      - rubocop-i18n
+      - rubocop-rspec
+
+appveyor.yml:
+  delete: true
```

Comment:
We're deleting appveyor because we guard against running on windows? How do we know those guards work / won't blow up a Puppet run?

Reply:
It was set like this before, i alphabetised the file, hence the stink.

Reply:
Fair! Worth noodling on for a future ticket, maybe, but I'm not blocking this one.